frontend/package.json

```diff
@@ -24,7 +24,8 @@
     "bnc-onboard": "^1.13.0",
     "core-js": "^3.6.5",
     "ethers": "^5.5.3",
-    "quasar": "^1.0.0"
+    "quasar": "^1.0.0",
+    "vue-i18n": "v8"
```

Comment:
Huh, what's with the weird version "number" here?

Reply:
Addressed this with specific version.